DEPENDENCIES.rst

```diff
@@ -141,15 +141,15 @@ Installing Dependencies (Mac OSX)
 Cyclus archetype development is not fully supported on Mac.  Nonetheless,
 because there are some use cases which require installation from source, we have
 compiled a list of instructions that should be successful.  Use a Mac platform
-at your own risk, we strongly recommend sticking to Linux for development.
+at your own risk; we strongly recommend Linux as a primary development environment.
 
-This guide assumes that the user has root access (to issue sudo commands) and
+This guide assumes that the user has root access (to issue `sudo` commands) and
 access to a package manager or has some other suitable method of automatically
 installing established libraries. This process was tested using a fresh install
-of Yosemite 10.10.2 using macports as the package manager.  Macports installs
-packages in /opt/local.  If installing to a different location,
-(i.e. /usr/local) change paths in the following instructions accordingly. If
-you use homebrew, try the following instructions with brew commands in place of
+of Yosemite 10.10.2 using `macports` as the package manager.  Macports installs
```

Comment:
it should be working at least on:
- OS X 10.9 Mavericks (not tested)
- OS X 10.10 Yosemite (tested by @mbmcgarry) 
- OS X 10.11 El Captain (tested by @baaaaam)
- OS X 10.12 Sierra (not tested)
previous version might be working too